Taskfile.yml

```
@@ -0,0 +1,191 @@
+
+  develop:
+    desc: Starts the Django Application, and background workers in a debug state
+    dir: .
+    cmds:
+      - echo "Completing Pre-Start Environment Check For Development..."
+      - source .envrc && PYTHONPATH="./core:{{ .TASKFILE_DIR }}/applications/API:{{ .TASKFILE_DIR }}/applications/graphQL:{{ .TASKFILE_DIR }}/applications/frontend:{{ .TASKFILE_DIR }}/common:{{ .TASKFILE_DIR }}/run:{{ .TASKFILE_DIR }}" doppler run  -- python3 .venv/bin/gunicorn -t 1 -w 1 --timeout 120 --worker-class=sync core.wsgi:application --bind 127.0.0.1:{{.CLI_ARGS}} --reload
+    interactive: false
+    silent: false
+
+  install:
+    deps: [ venv ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)